gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-fetch-configs.groovy

```diff
@@ -29,22 +29,6 @@ pipeline {
                 ])
             }
         }
-        stage('Fetch portal config') {
```

Comment:
Since the data-portal exposes the gitops.json, we can get it easily without getting into the VM. That's what the tests currently do [here](https://github.com/uc-cdis/gen3-code-vigil/blob/d368684/gen3-integration-tests/utils/gen3_admin_tasks.py#L15). The gitops.json file fetched by this job was never used